frontend/src/components/EditView.tsx

```diff
@@ -1,14 +1,17 @@
 import React from 'react'
 import MonacoEditor from './MonacoEditor'
 import { useSelector } from 'react-redux'
-import { RootState } from '../store/store'
+import { RootState, RepoFile } from '../types'
 
-interface RepoFile {
-    filename: string,
-    content: string,
+interface Props {
+    location: Location
 }
 
-const EditView = ({ location }: any) => {
+interface Location {
+    search: string
+}
```

Comment:
Location type can and should be imported from React-router history:
`import {  Location } from 'history'`